src/sentry/tasks/members.py

```diff
@@ -1,72 +0,0 @@
-from django.urls import reverse
```

Comment:
I think you need to remove `sentry.tasks.members` from `server.py` too cause this is making stuff angery